grails-app/assets/javascripts/asinha/payer-create.js

```diff
@@ -0,0 +1,95 @@
+function PayerCreate() {
+    
+    this.reference = $("#payer-create-container");
+    var addressInputReference = this.reference.find("#address").get(0);
+    var addressNumberInputReference = this.reference.find("#addressNumber").get(0);
+    var emailInputReference = this.reference.find("#email").get(0);
+    var provinceInputReference = this.reference.find("#province").get(0);
+    var cityInputReference = this.reference.find("#city").get(0);
+    var stateInputReference = this.reference.find("#state").get(0);
+    var cepInputReference = this.reference.find("#cep").get(0);
+    var errorMessageReference = this.reference.find("#error").get(0);
+    var searchCep;
+    var _this = this;
+    
+    this.init = function() {
+        bindForm();
+        bindCep();
+        bindEmail();
+        searchCep = new SearchCep();
+    };
+
+    var fillForm = (address ) => {
```

Comment:
Tem mais espaços em branco
```suggestion
    var fillForm = (address) => {
```